docs/semgrep-supply-chain/ignoring-deps.md

```diff
@@ -3,54 +3,48 @@ slug: ignoring-lockfiles-dependencies
 append_help_link: true
 description: "Prevent unwanted noise when scanning for dependency vulnerabilities by ignoring lockfiles or code files."
 tags:
-    - Semgrep Supply Chain
-title: Ignoring lockfiles and dependencies
+ - Semgrep Supply Chain
+title: Ignore lockfiles and dependencies
 hide_title: true
 ---
 
-# Ignoring lockfiles and dependencies
-
-
-There are several methods to restrict code files or lock files from generating dependency findings.
-
-Create a `.semgrepignore` file in your repository's root directory and define code files and lock files to ignore. For more information, see [Ignoring files, folders, and code](/ignoring-files-folders-code/#define-ignored-files-and-folders-in-semgrep-appsec-platform).
-
-Refer to the following table to see which goal best suits your need:
-
+# Ignore lockfiles and dependencies
 
+You can restrict code files or lockfiles from generating open source security findings. To do so, you must [create a `.semgrepignore` file in your repository's root directory](/ignoring-files-folders-code/#define-ignored-files-and-folders-in-semgrep-appsec-platform) and define code files and lock files to ignore. The file paths you provide in your `.semgrepignore` file depend on the option that best suits your organization's needs:
```

Comment:
(noting the use of "open source security findings" for when we discuss this with Sales/SSC team)

Reply:
checking w/ KK and Misha!